table-data-gateway/src/main/java/com/iluwatar/tabledatagateway/Person.java

```diff
@@ -0,0 +1,122 @@
+package com.iluwatar.tabledatagateway;
+
+/**
+ * The type Person.
+ */
+public class Person {
+
+  private int id;
+  private String firstName;
+  private String lastName;
+  private String gender;
+  private int age;
```

Comment:
Utilize Lombok to reduce boilerplate. See https://github.com/iluwatar/java-design-patterns/wiki/18.-Lombok